static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -70,8 +72,19 @@ export function ReplayCell({
     },
   };
 
+  const setTrackingEvent = () => {
```

Comment:
The method is returning a string, not accepting any params. It would be better to prefix it with `get*` instead of `set*`

But don't worry about this until after looking at the other comment in `replayAnalyticsEvents.tsx`